src/imageio.cs

```diff
@@ -2356,4 +2368,17 @@ interface CGImageAuxiliaryDataInfo {
 		NSData Data { get; set; }
 		NSDictionary DataDescription { get; set; }
 	}
+
+	[Partial]
+	[Mac (10, 15), iOS (13, 0), TV (13, 0), Watch (6, 0)]
+	interface CGImageAnimationOptions {
```

Comment:
where is `CGImageAnimationOptions` used ?
It looks like it should be a strong dictionary and have overload in some new API

Reply:
I don't see it used anywhere - the tests and sample pass in `null` for `NSDictionary options` and I don't see anything in the diffs. I can still rewrite it as a strong dictionary if you think that's the right thing to do.

Reply:
yes, we don't have to expose the `NSString` keys alone (whenever possible)
a strong dictionary allows us to hide the keys and make the API easier to use
header (or web) documentation might be helpful (@mandel-macaque already commented on them, and how they should look in a strong dictionary)

Reply:
Added `[StrongDictionary]`.

Based on the diff:

```
+/* The value of this key overrides the `delay time' specified by the image
+ * Value is a CFNumber of kCFNumberDoubleType.
+ */
+IMAGEIO_EXTERN const CFStringRef kCGImageAnimationDelayTime  IMAGEIO_AVAILABLE_STARTING(10.15, 13.0);
```

DelayTime is of type double:
`double DelayTime { get; set; }`

^ FYI @mandel-macaque